addons/website_slides/controllers/mail.py

```diff
@@ -76,11 +76,10 @@ def mail_update_message(self, res_model, res_id, message, message_id, redirect=N
             })
         channel = request.env[res_model].browse(res_id)
         return {
-            'message_id': message.id,
-            'message': html2plaintext(message.body),
-            'rating_value': message.rating_value,
+            'default_message_id': message.id,
+            'default_message': html2plaintext(message.body),
+            'default_rating_value': message.rating_value,
             'ratingAvg': channel.rating_avg,
-            'rating_count': channel.rating_count,
-            'attachment_ids': message.attachment_ids,
+            'ratingTotal': channel.rating_count,
             'default_attachment_ids': message.attachment_ids.sudo().read(['id', 'name', 'mimetype', 'file_size', 'access_token'])
```

Comment:
Again: don't mix and match snake_case and camelCase, stick to snake case. And again, there is no reason here to prefix the variable names with default_